src/core/ext/client_channel/client_channel_factory.h

```diff
@@ -71,6 +73,14 @@ struct grpc_client_channel_factory_vtable {
                                          const grpc_channel_args *args);
 };
 
+/** Return a channel argument containing \a targets_info. */
+grpc_arg grpc_channel_targets_info_create_channel_arg(
```

Comment:
I'm still not sure that this code belongs in the client_channel_factory module, since it's not used by anything that directly creates or uses a client channel factory.  It might be better to put it in the grpclb module.